ifoche/malariapp
# Bug/fixed refresh server record and getting logged user

### :pushpin: References
* **Issue:** close #2321 
* **Related pull-requests:** 

### :tophat: What is the goal?

Solve bugs open 1.4.3 from 1.4.2, show login instead of dashboard and toolbar in dashboard not show logo and name

### :memo: How is it being implemented?

- Does not get logged user filtering by username because on the last version is null
- Fixed a null pointer exception retrieving null server from db

### :boom: How can it be tested?

- **Use case 1:** install 1.4.2 and then install 1.4.3 then should navigate to dashboard
- **Use case 2:** install 1.4.2 and then install 1.4.3 then should navigate to dashboard and show logo and server name

### :floppy_disk: Requires DB migration?


### :art: UI changes?
